app/controllers/issue_templates_controller.rb

```diff
@@ -5,11 +5,10 @@ class IssueTemplatesController < ApplicationController
   helper :issues
   include IssuesHelper
   menu_item :issues
-  before_filter :find_object, :only => [ :show, :edit, :destroy ]
+  before_filter :find_object, only: [ :show, :edit, :destroy ]
   before_filter :find_user, :find_project, :authorize, 
-    :except => [ :preview, :move_order_higher, :move_order_lower, 
-                 :move_order_to_top, :move_order_to_bottom, :move ]
-  before_filter :find_tracker, :only => [ :set_pulldown ]
+    except: [ :preview, :move_order_higher, :move_order_lower, :move_order_to_top, :move_order_to_bottom, :move ]

```

Comment:
__[rubocop]__
* Align the parameters of a method call if they span more than one line. [:ref](https://github.com/bbatsov/ruby-style-guide#no-double-indent)
* Space inside square brackets detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-spaces-braces)
* Line is too long. [113/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)